README.md

```diff
@@ -0,0 +1,90 @@
+
+A hastily-proposed RFC can hurt its chances of acceptance. Low quality proposals, proposals for previously-rejected features, or those that don't fit into the near-term roadmap, may be quickly rejected, which can be demotivating for the unprepared contributor. Laying some groundwork ahead of the RFC can make the process smoother.
+
+Although there is no single way to prepare for submitting an RFC, it is generally a good idea to pursue feedback from other project developers beforehand, to ascertain that the RFC may be desirable; having a consistent impact on the project requires concerted effort toward consensus-building.
+
+The most common preparations for writing and submitting an RFC include talking the idea over on our [Linkerd Slack, #contributors](https://slack.linkerd.io), or discussing the topic on our [CNCF developer mailing list](https://lists.cncf.io/g/cncf-linkerd-dev). You may file issues on this repo for discussion, but these are not actively looked at by the teams.
+
+As a rule of thumb, receiving encouraging feedback from long-standing project developers is a good indication that the RFC is worth pursuing.
+
+## What the process is
+
+[what the process is]: #what-the-process-is
+
+In short, to get a major feature added to Linkerd, one must first get the RFC merged into the RFC repository as a markdown file. At that point the RFC is "active" and may be implemented with the goal of eventual inclusion into Linkerd.
+
+* Fork the [RFC repository](https://github.com/linkerd/rfc) * Copy `rfc-template.md` to `active/my-contribution.md` (where "my-contribution" is
+
+ descriptive.).
+
+* Fill in the RFC. Put care into the details: RFCs that do not present convincing motivation, demonstrate lack of understanding of the design's impact, or are disingenuous about the drawbacks or alternatives tend to be poorly-received.
+
+* Submit a pull request. As a pull request the RFC will receive design feedback from the larger community, and the author should be prepared to revise it in response.
+
+* Each pull request will be labeled with the most relevant reviewer, who will lead its triage.
+
+* Build consensus and integrate feedback. RFCs that have broad support are much more likely to make progress than those that don't receive any comments. Feel free to reach out to the RFC assignee in particular to get help identifying stakeholders and obstacles.
+
+* The team will discuss the RFC pull request, as much as possible in the comment thread of the pull request itself. Offline discussion will be summarized on the pull request comment thread.
+
+* RFCs rarely go through this process unchanged, especially as alternatives and drawbacks are shown. You can make edits, big and small, to the RFC to clarify or change the design, but make changes as new commits to the pull request, and leave a comment on the pull request explaining your changes. Specifically, do not squash or rebase commits after they are visible on the pull request.
```

Comment:
@admc What are your thoughts on the directory layout in this repo? Will we add an `accepted/` directory where active RFCs are moved?

Reply:
I separated the process into two steps, I don't have a strong opinion, but it seems like for a first try, having the whole conversation in one file will be easy to manage.